client/cgo/cgotest.go

```diff
@@ -74,202 +33,64 @@ func byteSliceToCcharPtr(buf []byte) *C.char {
 	return (*C.char)(ptr)
 }
 
-func testVerifyVrf(t *testing.T) {
-	sk, err := vrf.GenerateKey(nil)
-	if err != nil {
-		t.Fatal(err)
-	}
-	alice := []byte("alice")
-	aliceVRF, aliceProof := sk.Prove(alice)
-	pk, ok := sk.Public()
-	if !ok {
-		t.Fatal("Couldn't obtain public key!")
-	}
-	if v := C.testVerifyVrf(byteSliceToCucharPtr(pk), C.int(len(pk)),
-		byteSliceToCucharPtr(alice), C.int(len(alice)),
-		byteSliceToCucharPtr(aliceVRF), C.int(len(aliceVRF)),
-		byteSliceToCucharPtr(aliceProof), C.int(len(aliceProof))); v != 1 {
-		t.Error("cgoVrfVerify failed")
-	}
-}
-
-func testVerifySignature(t *testing.T) {
-	key, err := sign.GenerateKey(nil)
+func newDirectory(t *testing.T, useTBs bool) (*protocol.ConiksDirectory, sign.PublicKey) {
```

Comment:
> Separately, why do we call it InitDirectory instead of NewDirectory?

Ahh, I don't remember the reason. I will change it to `NewDirectory`.

@masomel What do you think about this?
> Should you be exporting the one from /protocol?
  
> I want to put testutil.go into /protocol/testutil package but it causes a import cycle.